src/sentry/models/organization.py

```diff
@@ -240,6 +240,10 @@ def __str__(self):
 
     snowflake_redis_key = "organization_snowflake_key"
 
+    def save_with_update_outbox(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+        Organization.outbox_for_update(self.id).save()
```

Comment:
We might end up generating lots of no-op outboxes messages this way. Flags get updated reasonably often and aren't required in the mapping records.

Perhaps we can find a way to conditionally generate outboxes based on the attributes that have changed? We can solve that later if we're able to identify redundant operations causing overhead.

Reply:
Yeah this was on the back of my mind as well. Although it would require an additional query, we could create a new method that takes an `Organization` which queries + diffs the old model against the provided instance, and could return an `Option[RpcOrganizationMappingUpdate]` only if any of the serializable fields have been modified.

Either that or some form of shim layer on the model itself that caches a copy of the initial member variables that flushes on save?

Reply:
To be totally, 100% honest, I'm more in favor of generating no-op outboxes than leaving correctness to chance.

Reply:
That's fair, and our update outbox backpressure should be capped with a worse case of the number of orgs thanks to coalescing right?

Reply:
> To be totally, 100% honest, I'm more in favor of generating no-op outboxes than leaving correctness to chance.

That's fair. Django doesn't make it easy to determine which fields have changed on a record either. Let's capture all the changes and if/when org updates are being expensive or high volume we can revisit.